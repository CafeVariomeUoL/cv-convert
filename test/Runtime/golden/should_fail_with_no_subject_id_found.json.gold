Error on line 0: subject_id is required