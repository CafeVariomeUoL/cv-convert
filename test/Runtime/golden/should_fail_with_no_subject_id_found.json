{
  "name": "Sam",
  "age": 25,
  "heart_rate": [110, 98, 123],
  "blood_pressure": [{"sys":95, "dia": 65}, {"sys":105, "dia": 72}, {"sys":98, "dia": 68}]
}